src/sentry/issues/search.py

```diff
@@ -235,8 +239,10 @@ def _update_profiling_search_filters(
     for sf in search_filters:
         # XXX: we replace queries on these keys to something that should return nothing since
         # profiling issues doesn't support have stacktraces
-        if sf.key.name in ("notHandled", "error.handled"):
-            updated_filters.append(SearchFilter(SearchKey("1"), "=", SearchValue("2")))
+        if sf.key.name in ("error.unhandled", "error.handled"):
```

Comment:
Does this mean `notHandled` won't be passed anymore? Also nit: "support" should be "supported" in the error message string.

Reply:
Yeah, queries like `error.handled` and `error.unhandled` will just throw an error and not query the issue platform dataset. As far as I can tell, these queries looks into the stacktrace data and in the context of profiling issues there are no exceptions.

This might change in the future though and we'll need to do some more fine tuning so we don't block valid queries, but for now I think rejecting any queries with these filters makes sense.

Reply:
To be clear, this change will reject any queries with the above filters on the issue platform dataset instead of replacing the condition with a 'dummy' condition.